test/integration-serial/serial_deploy_test.rb

```diff
@@ -453,18 +453,18 @@ def test_cr_failure_with_arbitrary_rollout_conditions
   end
 
   def test_deploying_crs_with_invalid_crd_conditions_fails
-    # Since CRDs are not always deployed along with their CRs and kubernetes-deploy is not the only way CRDs are
+    # Since CRDs are not always deployed along with their CRs and krane is not the only way CRDs are
     # deployed, we need to model the case where poorly configured rollout_conditions are present before deploying a CR
-    KubernetesDeploy::DeployTask.any_instance.expects(:validate_resources).returns(:true)
+    Krane::DeprecatedDeployTask.any_instance.expects(:validate_resources).returns(:true)
```

Comment:
Why not stub on `Krane`here? The deprecated task is a very temporary implementation detail, so I'm wondering why it should be referenced by anything at all other than its (temporary) subclasses.

Reply:
Hmmmm, I made it use public interface again... do you still think we should stub on `Krane` here?

Reply:
Actually I don't think you can use `Krane` here, since the integration tests are still using `KubernetesDeploy::DeployTask` via the helper.